graphql-dgs-spring-webmvc/src/main/kotlin/com/netflix/graphql/dgs/mvc/DgsRestController.kt

```diff
@@ -64,10 +61,10 @@ import org.springframework.web.multipart.MultipartFile
  */
 
 @RestController
-class DgsRestController(private val schemaProvider: DgsSchemaProvider, private val dgsQueryExecutor: DgsQueryExecutor) {
+class DgsRestController(private val dgsQueryExecutor: DgsQueryExecutor) {
     val logger: Logger = LoggerFactory.getLogger(DgsRestController::class.java)
 
-    @RequestMapping("/graphql", produces = ["application/json"])
+    @RequestMapping("\${dgs.graphql.path:/graphql}", produces = ["application/json"])
```

Comment:
I like detaching the definition of default values from the place where they are used. This can be done with an EnvironmentPostProcessor that loads a _default properties file_. 

That said, I think this should be considered as a separate pr.

Reply:
Thats a good idea especially when the number of properties grows and you can see all of the defaults in a single place. 

The one area where this is, not in conflict, but would need to be agreed upon is the use of `@DefaultValue` in the `@ConfigurationProperties`. It would be confusing having defaults in the config props and the defaults file. We could just make a guideline to not use `@DefaultValue` in config props though.